src/fmi_export/fmuBuilder.cpp

```diff
@@ -108,34 +108,20 @@ int fmuBuilder::Initialize (int argc, char *argv[])
 }
 using namespace boost::filesystem;
 /** helper function to copy a file and overwrite if requested*/
-bool testCopyFile (path source, path dest, bool overwrite = false)
+bool testCopyFile (path const& source, path const& dest, bool overwrite = false)
 {
-    if (!exists (dest))
+    copy_option option = copy_option::fail_if_exists;
+    if (overwrite) { option = copy_option::overwrite_if_exists; }
+
+    try
     {
-        try
-        {
-            copy_file (source, dest);
-            return true;
-        }
-        catch (filesystem_error &e)
-        {
-            return false;
-        }
+        copy_file (source, dest, option);
```

Comment:
copy_file does existence/overwrite handling already: just give it the right `copy_option`